### PR TITLE
Add [Feature] Inform / Donate button #2841

### PR DESCRIPTION
**Pull request by Tirokk**
> [Ruben80](https://github.com/Ruben80) **Authored by [Ruben80](https://github.com/Ruben80)**
_<time datetime="2020-02-21T20:01:40Z" title="Friday, February 21st 2020, 9:01:40 pm +01:00">Feb 21, 2020</time>_
_Closed <time datetime="2020-03-25T08:54:44Z" title="Wednesday, March 25th 2020, 9:54:44 am +01:00">Mar 25, 2020</time>_
---

<a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
          moderation@human-connection.org
        </a>

## 🍰 Pullrequest

Change label to "Inform / Donate"
On button press link to https://human-connection.org

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: webapp/pages/code-of-conduct.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
       <p>
         {{ $t('code-of-conduct.get-help') }}
-        <a class="hc-editor-link-blot" href="moderation@human-connection.org" target="_blank">
+        <a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
```
Comment (Tirokk) [inline webapp/pages/code-of-conduct.vue:28]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T11:52:25Z" title="Saturday, February 22nd 2020, 12:52:25 pm +01:00">Feb 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,7 +25,7 @@
 
       <p>
         {{ $t('code-of-conduct.get-help') }}
-        <a class="hc-editor-link-blot" href="moderation@human-connection.org" target="_blank">
+        <a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
```


Hey @Ruben80 it looks as if this branch introduces the same changes as #3133. Probably you wanted to commit/push something else?

**Comment by Tirokk** [comment]
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-25T08:54:44Z" title="Wednesday, March 25th 2020, 9:54:44 am +01:00">Mar 25, 2020</time>_

---


gonna close this. See :point_up_2:  @Ruben80

**Review by Mogge — CHANGES_REQUESTED** [review]
duplicate

I think this is a duplicate. @Ruben80 if you need help with git, consider to join our [community chat](https://human-connection.org/discord/) where we have enough people to help you update this branch with the relevant code.

**Review by Mogge — CHANGES_REQUESTED** [review]
hey @Ruben80 , actually this PR has no chance of passing our build because we use some encrypted variables and these are not made available to forks for security reasons. See https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

Could you please put in a PR directly to the parent repo?

**Pull request closed without merging**